src/jupyterhub_config/custom_spawner.py

```diff
@@ -206,7 +206,8 @@ def _configure_notebook_dir(self, username: str, user_dir: Path):
         """
         if self.user.admin:
             self.log.info(f'Admin user detected: {username}. Setting up admin workspace.')
-            self.notebook_dir = '/cdm_shared_workspace'
+            # global users' home directory
+            self.notebook_dir = str(user_dir.parent)
```

Comment:
So basically admin notebooks are all shared and live in /home and user notebooks live in /home/{user}? And users can read /home/{user} but not home?

Reply:
I think `str('/')` makes more sense for admin user.

Reply:
or /root

Reply:
or /admin

Reply:
👍 just changed it to '/' for now. I think most majority users won't be admin.

Reply:
That means any notebooks they create won't be persisted, I assume

Reply:
yea yea. I think they (or most likely just us) just need to remember to create notebook in the `jupyterhub/users_home` dir.